.github/workflows/build-uc2.yml

```diff
@@ -217,7 +217,7 @@ jobs:
           #export CC=i686-w64-mingw32-gcc
           export AR=gcc-ar
           export RANLIB=gcc-ranlib
-          export CFLAGS="-m32"
+          export CFLAGS="-m32 -static"
```

Comment:
Any reason for these? I remember it's for removing some runtime libraries dependency?